index.bs

```diff
@@ -2708,6 +2716,17 @@ The [=remote end steps=] with |session| and |command parameters| are:
       [=user context to accept insecure certificates override map=][|user context|]
       to |acceptInsecureCerts|.
 
+1. If |command parameters| [=map/contains=] "<code>proxy</code>":
+
+   1. Let |proxy configuration| be |command parameters|["<code>proxy</code>"].
+
+   1. If the |proxy configuration| cannot be configured return [=error=] with
+      [=error code=] [=invalid argument=].
```

Comment:
I think "if |configuration" cannot be configured" is a bit confusing, even though it's close to the language in WebDriver. Maybe:

```suggestion
   1. If the [=remote end=] is unable to configure proxy settings per [=user context=], 
       or is unable to configure the proxy with |proxy configuration|, return  [=error=] with
       [=error code=] [=unsupported operation=].
```